lib/engines/hcktest/platforms/jtd.json

```diff
@@ -0,0 +1,59 @@
+{
+  "discriminator": "setupmanager",
+  "mapping": {
+    "physhck": {
+      "properties": {
+        "name": { "type": "string" },
+        "kit": { "type": "string" },
+        "clients": {
+          "properties": {
+            "c1": { "ref": "physhck_client" }
+          },
+          "optionalProperties": {
+            "c2": { "ref": "physhck_client" }
+          }
```

Comment:
hcktest assumes there are two clients so `PhyWin2016.json` and `PhyWin2019.json` should be fixed instead.

Reply:
hcktest should not assume this. Different testing scenarios expect different machine counts. Our physical HLK has only one client but for the Server Stress SVVP test, we need three clients.

Reply:
It makes sense.